blocks/editable/patterns.js

```diff
@@ -49,6 +48,10 @@ export default function( editor ) {
 		canUndo = null;
 	} );
 
+	editor.on( 'pastepostprocess', () => {
+		setTimeout( () => searchFirstText( pastePatterns ) );
```

Comment:
With the `setTimeout` here (and the more recent `inline` handling), we need to make sure that the editor still exists in `searchFirstText`, since because it is called asynchronously, the editor might be removed in the time since scheduled. I discovered this in my own work (caused by an unrelated bug), but was difficult to track down the cause of the error'ing that occurs here.

Reply:
Yeah, I noticed some issues too in another PR. While this specific line has been removed, it's still an issue in the keydown event callback.